include/kframework/abi.md

```diff
@@ -154,7 +159,8 @@ where `F1 : F2 : F3 : F4` is the (two's complement) byte-array representation of
     rule #generateSignatureArgs(TARGA:TypedArg, .TypedArgs)            => #typeName(TARGA)
     rule #generateSignatureArgs(TARGA:TypedArg, TARGB:TypedArg, TARGS) => #typeName(TARGA) +String "," +String #generateSignatureArgs(TARGB, TARGS)
 
-    syntax String ::= #typeName ( TypedArg ) [function, total]
+    syntax String ::= #typeName ( TypedArg  )                            [function, total]
+                    | #typeName ( TypedArgs )                            [function, total]
  // ----------------------------------------------------------
```

Comment:
formatting
```suggestion
    syntax String ::= #typeName ( TypedArg  ) [function, total]
                    | #typeName ( TypedArgs ) [function, total]
 // -----------------------------------------------------------
```

Reply:
Thanks. because `#tuple` now just rewrites to a `TypedArgs` I just removed that `#typeName` rewrite rule.